src/main/scala/com/ubirch/util/elasticsearch/EsSimpleClient.scala

```diff
@@ -37,7 +37,7 @@ trait EsSimpleClientBase extends StrictLogging {
   implicit val ec: ExecutionContext = scala.concurrent.ExecutionContext.global
 
   private val esClient: RestHighLevelClient = EsHighLevelClient.client
-  private val maxRetries: Int = EsHighLevelConfig.maxRetries
+  private[elasticsearch] val maxRetries: Int = EsHighLevelConfig.maxRetries
```

Comment:
Just one small question: Why make it private to the package and not just to the class?

Reply:
Because otherwise I cannot overwrite the value in the test class.